model/insertClasses/ToAdminTable.php

```diff
@@ -7,13 +7,11 @@ class ToAdminTable {
     public $password;
     
     function __construct(
-            $id,
```

Comment:
`id` field should be removed because this class is also used to add an Admin. When we add an admin, the object passed will not have an id. The update method will receive a separate parameter for `id` as you implemented.

Reply:
This is showing that it was removed already.